abuild.in

```diff
@@ -1803,6 +1829,50 @@ openrc() {
 	default_openrc
 }
 
+default_bashcomp() {
+	depends=""
+	pkgdesc="Bash completions for $pkgname"
+	install_if="$pkgname=$pkgver-r$pkgrel bash-completion"
+
+	cd "$pkgdir" || return 0
+	for i in $(find usr/share/bash-completion/completions etc/bash_completion.d -type f 2>/dev/null); do
+		install -Dm0644 "$i" "$subpkgdir"/usr/share/bash-completion/completions/"$(basename $i)"
```

Comment:
Subpackage functions usually use `mkdir` + `mv`, like this you end up with a copy in the main package *and* in the completion package.
This applies to all 3 functions.

Reply:
fixed